tests/intern.js

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ // Learn more about configuring this file at <https://github.com/theintern/intern/wiki/Configuring-Intern>.
+// These default settings work OK for most people. The options that *must* be changed below are the
+// packages, suites, excludeInstrumentation, and (if you want functional tests) functionalSuites.
+define(['intern/lib/args'], function (args) {
+  'use strict';
+
+  return {
+    // Chronicle configuration
+    chronicle: {
+      url: 'http://localhost:8080/'
```

Comment:
Yeah it should. It doesn't like trying to do a regular old node `require` in this context, so I need to figure that out.